### PR TITLE
Fixed Bug-8431 Old solution Actions are pointing to LiteDB

### PR DESCRIPTION
**Pull request by arvindgh**
Added Check to get already saved ActOutDataSourceConfig params

### CODE REVIEW

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 4
           

Complexity increasing per file
==============================
- Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237&bid=16013653)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs [hunk 1]
```diff
@@ -48,42 +49,98 @@ public  static partial class Fields
...
         [IsSerializedForLocalRepository]
         public string DSTable { get; set; }
-        
-        [IsSerializedForLocalRepository]
-        public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
 
-        [IsSerializedForLocalRepository]
-        public string OutputType { get; set; }
+        //[IsSerializedForLocalRepository]
+        //public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
```
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs [hunk 2]
```diff
@@ -48,42 +49,98 @@ public  static partial class Fields
...
             set
             {
-                this.OutputType = value;
+                this.DSName = value;
             }
         }
 
-        ObservableList<string> mPossibleValues = new ObservableList<string>();
-        public ObservableList<string> PossibleValues
+        //ObservableList<string> mPossibleValues = new ObservableList<string>();
```
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs [hunk 3]
```diff
@@ -18,28 +18,29 @@
...
-        public  static partial class Fields
+        public static partial class Fields
         {
             public static string DSName = "DSName";
             public static string DSTable = "DSTable";
-            public static string Active = "Active";
-            public static string OutputType = "OutputType";
-            public static string TableColumn = "TableColumn";
-            public static string PossibleValues = "PossibleValues";
+            //public static string Active = "Active";
```
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs [hunk 4]
```diff
@@ -48,42 +49,98 @@ public  static partial class Fields
...
-                OnPropertyChanged(Fields.PossibleValues);
+                if (name == "Active" || name == "OutputType" || name == "TableColumn")
+                {
+                    return true;
+                }
             }
+            return false;
+        }
+
+        public override void PostSerialization()
```
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 4
           

Complexity increasing per file
==============================
- Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237&bid=16013653)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs (hunk 1 above)
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs (hunk 2 above)
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs (hunk 3 above)
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

File: Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs (hunk 4 above)
Comment (ambarkhan) [inline Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs]: Codacy Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)

**Comment by arvindgh** [comment]
closing for now, will do new serialization after release

**Pull request closed without merging**